cdk/src/open-data-platform/app-plane/tileserver/tileserver.ts

```diff
@@ -0,0 +1,69 @@
+// Defines the tile server, which users will request to get vector map data.
+
+import { Construct } from 'constructs';
+import * as ecs from 'aws-cdk-lib/aws-ecs';
+import * as ecsPatterns from 'aws-cdk-lib/aws-ecs-patterns';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as rds from 'aws-cdk-lib/aws-rds';
+import * as secretsmanager from 'aws-cdk-lib/aws-secretsmanager';
+
+interface TileServerProps {
+  ecsCluster: ecs.Cluster;
+  postgisTileDatabase: rds.ServerlessCluster;
+  databaseUrlSecret: secretsmanager.ISecret;
+}
+
+export class TileServer extends Construct {
+  readonly ecsService: ecsPatterns.ApplicationLoadBalancedFargateService;
+
+  constructor(scope: Construct, id: string, props: TileServerProps) {
+    super(scope, id);
+
+    const { ecsCluster, postgisTileDatabase, databaseUrlSecret } = props;
+
+    // Use Fargate, which runs the container in a serverless way.
+    // TODO: consider using spot instances for lower cost: https://github.com/aws/aws-cdk/blob/master/design/aws-ecs/aws-ecs-fargate-capacity-providers.md
+    this.ecsService = new ecsPatterns.ApplicationLoadBalancedFargateService(this, 'Service', {
+      cluster: ecsCluster,
+      taskSubnets: {
+        subnetType: ec2.SubnetType.PRIVATE_WITH_NAT,
+      },
+      desiredCount: 1,
+      memoryLimitMiB: 1024,
+      cpu: 512,
```

Comment:
Do we know if these are standard? Or did we just put what BC has? Either way can we add a common on significance of these numbers?

Reply:
I don't have a ton of info, but I've added some.